VideoLocker/src/main/java/org/edx/mobile/module/serverapi/IResponse.java

```diff
@@ -0,0 +1,13 @@
+package org.edx.mobile.module.serverapi;
+
+import android.os.Bundle;
+
+/**
+ * Created by rohan on 2/6/15.
+ */
+public interface IResponse {
```

Comment:
Yeah, as I said in above comment, we want to detach the HTTP component so that it can be easily replaced with another alternative library.